src/QsCompiler/Core/NamespaceTransformation.fs

```diff
@@ -183,7 +206,7 @@ type NamespaceTransformationBase internal (unsafe) =
         let specializations = c.Specializations |> Seq.map this.dispatchSpecialization
```

Comment:
It would be convenient if it were possible to guarantee the order of specializations, at least that the body is always seen first. I realize this might be awkward, and it can be worked around, but being able to look at the body before the other specializations is sometimes very convenient in transformations.

Reply:
done